src/main/java/me/fallenbreath/yamlang/YamlangConvertor.java

```diff
@@ -71,4 +87,54 @@ private static String renameYaml2Json(String fileName)
 		String baseName = fileName.substring(0, fileName.length() - YAML_PREFIX.length());
 		return baseName + JSON_PREFIX;
 	}
+
+	private String yamlang2Jsonlang(String ymlContent)
```

Comment:
I think keeping `yamlang2Jsonlang` and `parseMap` inside the `Yamlang2JsonlangMapper` will be better, cuz these 2 methdods are only used in `Yamlang2JsonlangMapper`

Reply:
They can't be because
1. Non-static inner classes can't have static methods in Java 8.
2. Those methods can't be members of `Yamlang2JsonlangMapper` because they are called before `super`.